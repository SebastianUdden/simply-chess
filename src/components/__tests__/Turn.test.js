import React from "react"
import renderer from "react-test-renderer"
import Turn from "../Turn"

describe("Turn", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Turn playerTurn="white" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
