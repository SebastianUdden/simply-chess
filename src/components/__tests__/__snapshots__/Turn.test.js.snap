// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Turn renders correctly 1`] = `
<h2
  className="sc-AxjAm jWlVxK"
>
  <span
    className="sc-AxirZ gfQzQh"
    color="white"
  >
    <strong
      className="sc-AxiKw gEgewe"
      color="white"
      id="playerTurn"
    >
      White
    </strong>
    's turn
  </span>
</h2>
`;
